Sources/CoreCommands/Options.swift

```diff
@@ -270,11 +270,28 @@ public struct BuildOptions: ParsableArguments {
                 visibility: .hidden))
     public var xcbuildFlags: [String] = []
 
+    @Option(name: .customLong("Xhost-swiftc", withSingleDash: true),
+            parsing: .unconditionalSingleValue,
+            help: ArgumentHelp(
+                "Pass flag to swiftc build invocation targetting the host (manifest and plugins)",
```

Comment:
```suggestion
                "Pass a flag to `swiftc` build invocation for build-time executables (manifest and plugins)",
```

Reply:
I decided to leave out the change of "a flag" because no other `-X...` flag has that article in their help string. Addtionally instead of adding quotes around `swiftc` I changed to use "Swift compiler", which is the wording that is used in `-Xswiftc`.